packages/json-rpc/sm-provider/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixed
 
 - Changed build to show individual files on source map explorers [#567](https://github.com/polkadot-api/polkadot-api/pull/567)
+- patch dependencies
```

Comment:
Not important, but usually I don't put these changes, as I think the internal dependencies are always implied to be upgraded to the next one.

I only do if it's the only reason for a new version.

Reply:
good point! yeah, this makes sense, I will keep in in mind for the future, thanks @voliva !